src/metax_api/api/base/views/dataset_view.py

```diff
@@ -53,9 +54,13 @@ def get_queryset(self):
         if hasattr(self, 'queryset_search_params'):
             additional_filters.update(**self.queryset_search_params)
 
+        CS.set_if_modified_since_filter(self.request, additional_filters)
```

Comment:
I think this line is redundant. Line 62-63 already places query parameters from request to self.queryset_search_params, which is moved to additional_filters on line 55?

Reply:
If this line is removed, tests fail, which by itself means nothing of course. But it seems get_queryset method on line 45 is only called by list-operation GET (/datasets) and retrieve method on line 61 is called only by detail-operation GET (datasets/pid). So if row 57 was removed, If-Modified-Since header would not work for rest/datasets(/urn_identifiers) GET operations.

Reply:
OK